packages/compass-e2e-tests/tests/collection-schema-tab.test.ts

```diff
@@ -109,6 +109,54 @@ describe('Collection schema tab', function () {
     });
   }
 
+  describe('with the enableExportSchema feature flag enabled', function () {
+    beforeEach(async function () {
+      // TODO(COMPASS-8819): remove web skip when defaulted true.
+      skipForWeb(this, "can't toggle features in compass-web");
```

Comment:
Hmmm, I think you added support for this in https://github.com/mongodb-js/compass/pull/6547 already so you should be able to toggle it